job/common/src/main/java/alluxio/job/util/SerializationUtils.java

```diff
@@ -31,6 +31,11 @@
 public final class SerializationUtils {
   private SerializationUtils() {} // prevent instantiation
 
+  /**
+   * The identifier represents the starting position of the benchmark result.
+   */
+  public static final String BENCHMARK_RESULT_TAG = "BENCHMARK RESULT:";
```

Comment:
@yabola do you mind sharing which unit test would fail because of not having this header? We would like to remove this header and keep the output backcompat.

Reply:
I don't quite remember which unit test it is. When running in the packaged alluxio, sometimes the log has warn/error log output (for example, the list file is too long). As long as there is this part of the output, there will be problems with json parsing (similarly If you adjust the log level, that will also)